tests/unit/core/testErrorHandler.js

```diff
@@ -263,5 +263,15 @@ describe('errorHandler', () => {
       const handler = new ErrorHandler();
       expect(handler.hasError()).toBe(false);
     });
+
+    it('returns no component if it does not have an error', () => {
+      const handler = new ErrorHandler();
+      expect(handler.renderErrorIfPresent()).toBe(null);
+    });
+
+    it('returns a component if it has an error', () => {
+      const handler = new ErrorHandler({ capturedError: new Error() });
+      expect(handler.renderErrorIfPresent()).not.toBe(null);
```

Comment:
That is way better indeed.

I need to use some sort of Enzyme wrapper though or did I miss a testing lib that allows to write this that way?